src/etc/inc/util.inc

```diff
@@ -1142,6 +1147,11 @@ function is_portoralias($port) {
 	}
 }
 
+/* returns true if $port is a valid TCP/UDP port number or range ("<port>:<port>") or an alias thereof */
+function is_portorrangeoralias($port) {
+	return (is_portoralias($port) || is_portrange($port));
```

Comment:
In the 2nd call it only needs to check if it is a "portrange", because the 1st part has already verified if it is a "portoralias". So this will return true if it is any of those 3 things.
The order of evaluation came directly from its use in firewall_nat_out_edit.php
I guess there could be a function is_portalias() that separates out the "is_alias" and "is_port" stuff in "is_port_or_alias". Then we can have:
```
function is_portorrangeoralias($port) {
	return (is_port($port) || is_portrange($port) || is_portalias($port));
}
```